lib/server.js

```diff
@@ -505,6 +512,10 @@ WalletService.prototype.addAccess = function(opts, cb) {
   });
 };
 
+WalletService.prototype._clientSupportsBIP44 = function() {
+  return !!this.clientVersion && !(/^bw.-0\.[01]\./.test(this.clientVersion));
```

Comment:
I think it would be more clear to refactor this, to something like:
```
  if (!this.clientVersion) return false;
  if (this.clientVersion.major == 0 &&  this.clientVersion.minor <= 1) return false;
  return true;
```